.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,29 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend]: [Bug Name]"
```

Comment:
I think typically when we use brackets, the `:` is no longer necessary.

so i'd suggest format of `[backend] <Bug Name>`

Reply:
That makes sense, let me update.